.github/workflows/CI.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           environment-file: environment.yml
           create-args: >-
             lfortran=${{ env.LFORTRAN_VERSION }}
+            llvmdev=11.1.0
```

Comment:
This must become a different test.

We need to test that lfortran just works with the minimal environment without LLVM.

In a separate test, we will test that if LLVM is installed, then the `-g` option works.

Reply:
>This must become a different test.

Yes, sorry, my bad. I thought I created a separate test.